templates/repo/settings/protected_branch.tmpl

```diff
@@ -2,7 +2,11 @@
 	<div class="repo-setting-content">
 		<form class="ui form" action="{{.Link}}" method="post">
 			<h4 class="ui top attached header">
-				{{ctx.Locale.Tr "repo.settings.branch_protection" .Rule.RuleName}}
+				{{if .Rule.RuleName}}
```

Comment:
Does Gitea allow empty rule name or not?

* If yes: this check is not right
* If no, it's better to add "required" to the rule name input

Reply:
It's not allowed. However, when clicking the "New" button on the list page, it takes user to the "before" page mentioned above, and something like [''] appears in the header.

Reply:
What about this: `add "required" to the rule name input`?